djstripe/settings.py

```diff
@@ -150,6 +151,23 @@ def get_subscriber_model():
     return subscriber_model
 
 
+def get_api_version():
+    """Get the desired API version to use for Stripe requests."""
+    version = getattr(settings, 'DJSTRIPE_API_VERSION', None) or '2017-02-14'
+
+    if version == 'latest':
```

Comment:
Hmm, that's not completely true, but I agree that this might be confusing.  If the user specifies 'latest' here then the client API will actually use the Stripe version configured in their account.  I had actually started out with the assumption that 'latest' upstream does what it says on the tin (use the latest version), but it doesn't.  Either we agree on how to do this, or we remove it completely.  Thoughts?